userdocs/userdocs.docc/userdocs.md

```diff
@@ -0,0 +1,45 @@
+# vscode-swift
+
+@Metadata {
+   @TechnologyRoot
+}
+
+Language support for Swift in Visual Studio Code.
+
+@Metadata {
+    @DisplayName("vscode-swift")
+}
+
+This extension adds language support for Swift to Visual Studio Code, providing a seamless experience for developing Swift applications on all supported platforms. It supports:
```

Comment:
```suggestion
This extension adds language support for Swift to Visual Studio Code, providing a seamless experience for developing Swift applications. It supports:
```

Given that you can't really build iOS based apps (at least in my understanding) through VSCode, I'm not sure "all supported platforms" really fits.